package.json

```diff
@@ -22,6 +24,7 @@
         "@creativecommons/vocabulary": "^2020.11.3",
         "@creativecommons/vue-vocabulary": "^2020.11.3",
         "@sentry/vue": "^5.29.2",
+        "babel-plugin-transform-require-context": "^0.1.1",
```

Comment:
What is this plugin for? Should it be a `devDependency`?

Reply:
To support webpack build in Jest test cases. Yes, you are right. I should move it to dev-dependencies